documentation/modules/ROOT/pages/01-setup.adoc

```diff
@@ -6,6 +6,8 @@ include::_attributes.adoc[]
 
 The following CLI tools are required for running the exercises in this tutorial. Please have them installed and configured before you get started with any of the tutorial chapters.
 
+You don't have to have both `Minikube` and `Minishift`. Also, if you have an existing OpenShift cluster you can also use it.
```

Comment:
@aliok - wondering if still need this note, as the current documentation has split vanilla kubernetes and openshift instructions.

Reply:
ok, sure. but, still saying you need minishift is kind of misleading if a person is only interested in minikube path.

When I did the tutorial for the first time, I was thinking "oh, ok I need both minikube and minishift. probably later sections have something that's only working on OpenShift".

not a strong opinion, not a big thing. ok to drop this.